v3/plugins/scalameta/metals/1.9.5/meta.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+publisher: scalameta
+name: metals
+version: 1.9.5
+type: VS Code extension
+displayName: Scala (Metals)
+title: Scala language server with rich IDE features
+description: This plug-in provides support for Scala development via the Metals language server.
+icon: https://raw.githubusercontent.com/scalameta/metals-vscode/v1.9.5/images/logo.png
+repository: https://github.com/scalameta/metals-vscode
+category: Language
+firstPublicationDate: "2020-11-02"
+spec:
+  containers:
+    - image: "quay.io/gattytto/scala090:0.9.0"
```

Comment:
What's different about this image? It's probably best to update the side in che-sidecar-scala, no?

Reply:
I've created a ticket to fix the image https://github.com/che-dockerfiles/che-sidecar-scala/issues/10

Reply:
when [this one](https://github.com/eclipse/che-plugin-registry/pull/682/files) is merged you may use the new sidecar ID for quay.io/eclipse/che-sidecar-scala:0.9.0-xyz here @sideeffffect